src/contexts/AuthProvider/context.ts

```diff
@@ -7,8 +7,9 @@ export interface ContextProps {
   setCurrentUser: (data: any) => void;
   getCurrentUser: any;
   logout: () => void;
-  updateMyProfile: (editedUserProfile: EditableUserProfile) => void;
-  deleteMyProfileImage: () => void;
+  updateMyProfile: (editedUserProfile: EditableUserProfile) => Promise<void>;
+  updateMyProfileImage: (editedProfileImageFile: File) => Promise<void>;
+  deleteMyProfileImage: () => Promise<void>;
```

Comment:
`p1` Promise의 제네릭으로 void가 맞나요? 

```suggestion
  updateMyProfile: (editedUserProfile: EditableUserProfile) => Promise<any>;
  updateMyProfileImage: (editedProfileImageFile: File) => Promise<any>;
  deleteMyProfileImage: () => Promise<any>;
```

```m
"void" is a little like the opposite of any: the absence of having any type at all. You may commonly see this as the return type of functions that do not return a value
``` 

- 타입스크립트 공식 핸드북 void 설명
https://www.typescriptlang.org/docs/handbook/basic-types.html#void